src/datasources/documize.js

```diff
@@ -37,7 +37,9 @@ class DocumizeRestAPI extends RESTDataSource {
       keywords,
       ...searchOptions,
     });
-    return data.slice(0, limit).map(this.reduceDocumizeSearch.bind(this));
+    const uniqueSearchResult = _.uniqBy(data, 'documentId');
```

Comment:
so the documize search api is returning multiples of data?

Reply:
Yes, it will return exactly the same number of matches in the document. 
For example, if we search api, it appears 4 times, 3 times in the same document, then the documize search api will return that document 3 times.

Reply:
lame